src/component/About.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import styles from '../index.css';
-import {Form,Col,ItemLayout, Icon, Input, Button, Checkbox } from 'antd';
+import {Form,Col,ItemLayout, Icon, Input, Button, notification, Checkbox } from 'antd';
 import style from 'antd/dist/antd.css';
```

Comment:
there is no need to import this every where, it can be import once in index or app file
see documentation - https://ant.design/docs/react/introduce